docs/teams.md

```diff
@@ -7,30 +7,43 @@ layout: docs
 
 Many Tilt Cloud features are based on the concept of a Tilt team. Once you've signed in to Tilt Cloud, create a team by selecting the option at the top right, entering a team name, and clicking Create My Team. This will bring you to the team page of your new team. You are the team owner as a result. To navigate to an existing team that you own, hover over the Teams dropdown at the top right of the screen.
 
-You can also change the team name on this page. 
+You can also change the team name on this page.
 
 <figure>
     <img src="/assets/img/new-team.png" class="no-shadow" alt="New team">
 </figure>
 
 ## Associate team with a project
 
-Tilt is designed to have a single team be associated with a single Tiltfile of a project. As the team owner, after you've created a team, copy the team id (which is the string of text after `/team` in the URL, e.g. `27d7ad9c-53f8-4700-80b8-b217eeb8effg`), and enter it into your Tiltfile with the [`set_team( )` function](./api.html#api.set_team), for example: 
+Tilt is designed to have a single team be associated with a single Tiltfile of a project. As the team owner, after you've created a team, copy the team id (which is the string of text after `/team` in the URL, e.g. `27d7ad9c-53f8-4700-80b8-b217eeb8effg`), and enter it into your Tiltfile with the [`set_team( )` function](./api.html#api.set_team), for example:
 
 `set_team('27d7ad9c-53f8-4700-80b8-b217eeb8effg')`
 
-Commit the change to source control. 
+Commit the change to source control.
 
 ## Last seen versions
 
-You'll now see all folks who run Tilt with the updated Tilfile, appear on the team page, including when they last ran Tilt and the version they are using. Ask users to sign in to Tilt Cloud in Tilt for their name to appear on the team page. Otherwise, they will appear as anonymous.
+You'll now see all folks who run Tilt with the updated Tiltfile, appear on the team page, including when they last ran Tilt and the version they are using. Ask users to sign in to Tilt Cloud in Tilt for their name to appear on the team page. Otherwise, they will appear as anonymous.
 
 <figure>
     <img src="/assets/img/last-seen-versions.png" class="no-shadow" alt="Last seen versions">
 </figure>
 
+## Vet Tilt releases before your team upgrades
+
+Tilt's web UI will nudge users to upgrade when a new release is available.
+
+When running Tilt with a Tiltfile that specifies a team id, the default behavior is that users are only notified about Tilt releases once they're one week old.
```

Comment:
```suggestion
When running Tilt with a Tiltfile that specifies a team id, non-`Owner` users are only notified about Tilt releases once they're one week old.
```

Reply:
I included the phrase "default behavior" to get the broad idea across while indicating that there are exceptions. IIUC, your suggestion is trying to capture one of those exceptions (owners) but not another (manual promotions), while dropping the "default behavior" qualifier, which makes this statement not entirely true by itself.

Was your main goal to get the "non-Owner" exception in there? How would you feel about "the default behavior is that non-Owner users..."?

Reply:
The latest version looks good.